core-web/libs/sdk/experiments/src/lib/react/hooks/useExperiments.tsx

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Now based on what we talked last week, this library `@dotcms/experiments` should NOT BE a react library.

So from my perspective we should not merge this and create `@dotcms/experiments` as a JavaScript (TypeScript) library.

And the react one which probably will be a hook we can have that hook in `@dotcms/react`

And we could import that like

```javascript
import {useExperiments} from '@dotcms/react/hooks'
```

Or something like that.

Reply:
Yes, I agree. Let's go to close this PR.